src/main/java/fi/helsinki/cs/tmc/core/domain/submission/SubmissionResult.java

```diff
@@ -104,14 +102,6 @@ public void setError(String error) {
         this.error = error;
     }
 
-    public Validations getValidations() {
-        return validations;
-    }
-
-    public void setValidations(Validations validations) {
```

Comment:
I'm wondering whether these would actually be needed; I recall that this class is instantiated by a json library and server does return validation results also when exercises are configured to run validations (e.g checkstyle)